pom.xml

```diff
@@ -17,6 +17,22 @@
   </issueManagement>
 
   <inceptionYear>2010</inceptionYear>
+  
+  <developers>
+    <developer>
+      <id>chingor</id>
+      <name>Jeff Ching</name>
+      <email>chingor@google.com</email>
+      <organization>Google</organization>
+      <roles>
+        <role>Developer</role>
+      </roles>
+    </developer>
+  </developers>
+  <organization>
```

Comment:
remove this. There's already an organization on line 27

Reply:
removed